stable_baselines3/dqn/dqn.py

```diff
@@ -169,7 +169,12 @@ def _on_step(self) -> None:
         """
         self._n_calls += 1
         if self._n_calls % self.target_update_interval == 0:
-            polyak_update(self.q_net.parameters(), self.q_net_target.parameters(), self.tau)
+            included_names = ["weight", "bias", "running_"]
```

Comment:
I would rather pre-compute all the keys for batch norm when creating the policy `setup_model()`
and then do a hard copy of the statistics (because a linear interpolation may not make sense).
This would similar to synchronizing the observation normalization.
I think we can take a look at BYOL implementation (if I recall, there is a delayed target net and it has batch norm)

Reply:
Yeah the linear interpolation was my concern too. In my little project I used tau=0.9 with linear interpolation on running_mean/running_var and for reasons I don't know, it worked well. But I totally agree that a hard copy makes a lot more sense. Haven't read BYOL but I will take a look.

Reply:
Looking at BYOL implementation, they actually keep `training=True` for the target net (sounds a bit hacky, I need to ask if it is intented or a lucky oversight)

Reply:
Wow! I remember at first I also set `training=True` but that wasn't my intention. I specifically compared the running mean and var between the original net and my target net and they are quite different. The result was bad. This was how I noticed this batch norm problem at the beginning.

Reply:
> I would rather pre-compute all the keys for batch norm when creating the policy `setup_model()` and then do a hard copy of the statistics (because a linear interpolation may not make sense). This would similar to synchronizing the observation normalization. I think we can take a look at BYOL implementation (if I recall, there is a delayed target net and it has batch norm)

Just made the changes. Now I collected all batch norm parameters in `_setup_model()` and made `polyak_update` with `tau=1.0` on them. Let me know if this is what you meant.